package.json

```diff
@@ -6,33 +6,35 @@
     "start": "ng serve",
     "build": "ng build",
     "test": "ng test",
-    "lint": "ng lint",
+    "lint": "eslint --ext .ts . --quiet",
     "e2e": "ng e2e",
-    "watch-keyboard": "ng build angular-keyboard --watch",
-    "build-keyboard": "ng build angular-keyboard --configuration production",
+    "watch-keyboard": "npm run lint && ng build angular-keyboard --watch",
+    "build-keyboard": "npm run lint && ng build angular-keyboard --configuration production",
```

Comment:
We do not exactly have a pre-commit hook here, so I thought I'd place the linters before the build processes.

Reply:
sounds good.